App/screens/CalendarScreen.js

```diff
@@ -0,0 +1,61 @@
+import { StyleSheet } from 'react-native';
+import { Text, View } from '../components/Themed';
+import React, { Component, useState } from 'react';
+import { Calendar, LocaleConfig, Agenda, CalendarList } from 'react-native-calendars';
+
+LocaleConfig.locales['en'] = {
+  monthNames: [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December'
+  ],
+  monthNamesShort: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  dayNames: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+  dayNamesShort: ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'],
+  today: 'Today'
+};
+LocaleConfig.defaultLocale = 'en';
```

Comment:
wat gebeurt er met de calendar als je dit niet hebt gedefined? lijkt een beetje onnodige code op dit moment. omdat we maar 1 taal hebben